programs/bpf_loader/src/syscalls.rs

```diff
@@ -1247,58 +1247,6 @@ impl<'a> SyscallObject<BPFError> for SyscallInvokeSignedC<'a> {
     }
 }
 
-fn verify_instruction<'a>(
```

Comment:
Seems like this refactoring could be a separate precursor PR

Reply:
Yeah, was thinking of breaking that and the BankClient changes out into separate ones, didn't seem too important but could at risk of slowing down getting this into 1.4

Reply:
Yeah I guess vote for small/quick-to-review PRs in general.   Details get lost in big PRs

Reply:
Yup, did that a lot with the CPI stuff, tons a dependent little prs fell out of that.  Backporting a ton of little prs can create a long tail depending on how dependent they are on each other, lots of babysitting CI